curriculum/challenges/english/25-front-end-development/workshop-recipe-tracker/66fbcf750a62784cf11f5634.md

```diff
@@ -0,0 +1,133 @@
+---
+id: 66fbcf750a62784cf11f5634
+title: Step 10
+challengeType: 1
+dashedName: step-10
+---
+
+# --description--
+
+Time to test each of the functions. You can use any of the recipes for this, but go with `recipe1` for now.
+
+Create a `recipe1AverageRating`, `recipe1TotalIngredients`, and `recipe1DifficultyLevel` variables . For their values, call the function that corresponds to each variable name and pass in the appropriate `recipe1` property.
+
+Also, log each of the variables to the console to see what they look like.
+
+# --hints--
+
+You should create a `recipe1AverageRating` variable.
+
+```js
+assert.isNotNull(recipe1AverageRating)
+```
+
+Your `recipe1AverageRating` variable should be set to `getAverageRating(recipe1.ratings)`.
+
+```js
+assert.equal(recipe1AverageRating, getAverageRating(recipe1.ratings));
+```
+
+You should log `recipe1AverageRating` to the console with the text `"Recipe 1 Average Rating"` followed by the variable name.
+
+```js
+assert.match(code, /console\.log\((?:(`|"|')Recipe\s+1\s+Average\s+Rating\s*:\s*(`|"|')\s*\+\s*recipe1AverageRating|`Recipe\s+1\s+Average\s+Rating\s*:\s*\$\{recipe1AverageRating\}`)\);?/)
+```
```

Comment:
The directions don't mention logging the string followed by the variable name. But also, this didn't pass 

```md
console.log("Recipe 1 Average Rating",recipe1AverageRating) 
```

to make testing easier, we should just have them log the variables to the console which is what the directions say to do